StoreApp.BLL/Services/OrderService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+using System.Collections.Generic;
+
+namespace StoreApp.BLL.Services
+{
+    class OrderService : IOrderService
+    {
+        IUnitOfWork DataBase;
+        public OrderService(IUnitOfWork eof)
+        {
+            DataBase = eof;
+        }
```

Comment:
The methods must be splitted by using indents

Reply:
fixed